book/website/collaboration/inclusive-events.md

```diff
@@ -6,34 +6,40 @@
 It covers: 
 - planning and logistical considerations
 - guidance on how to evaluate and choose a location
-- considerations and requirement for developing a communications strategy
+- considerations and requirements for developing a communications strategy
 - suggestions for how to create an inclusive and accessible agenda
-- accessible technology requirements and reccomendations for hybrid events
+- accessible technology requirements and recommendations for hybrid events
 - and provides a checklist that can be tailored to your specific event
 
 
 (cl-inclusive-events-background)=
 ## Motivation and Background
 
-Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background. 
-This chapter is meant to serve as a resource to support and guide our community members with the tools to identify the opportunities and challenges in upholding this, and to approach event planning and design with accessibility and inclusion in mind.
+Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background and there has been an increased effort in many disciplines to involve members of the public and people with lived experience in research projects. 
 
-Because both internal resources and the events you run can range in size - from from small meetings to massive multi-day conferences - this chapter is not an exhaustive list and instead is meant to provide general guidance about how to design events that are inclusive for people with lived experience. 
+This chapter is meant to serve as a resource to support and guide researchers, organizers, and community members with the tools to identify the challenges and opportunities in organizing events that involve and include people with lived experience.
+
+We refer to "people with lived experience" throughout this chapter as a way to recognize and acknowledge individuals who have direct personal knowledge, or firsthand experience of a particular situation, condition, or topic. 
+It is a term often used to emphasise the importance of including the perspectives and insights of those who have had direct experience of a specific condition or situation. 
+In the context of event planning, the considerations need to cover both the disability requirements of your attendees and an identification of barriers that would be experienced by your specific attendees with lived experience. 
+For example, attendees who are immunocompromised, experience long-term health conditions, people with disabilities, and those with caring responsibilities would all be considered as people with lived experience, and would have slightly different accommodations needed to support their engagement in an event. 
```

Comment:
I think the comment I left on the prior version is still somewhat relevant on this to avoid conflating general accessibility & inclusion aspects with the "people with lived experience". 

A possible way of framing this would be that depending on the research topic the people with lived experience might be more likely to have accessibility requirements (e.g. when doing research on disabilities, health conditions, etc…) and that there might also be particular inclusion requirements that come from not being part of professional research (e.g. having different expectations, social networks, …)

Reply:
I drafted this sentence which can go on line 40. 

> Not all attendees with lived experience will have a disability, but if the focus of the experience you are running your event for is a health condition or disability, then you will need to consider both accessibility and inclusion requirements. 

I do feel like it would be more confusing to add that in on the landing page and it might be better served in a section of TTW which discusses PPIE in general. 

I would be happy making the assumption that people coming to this chapter for guidance on event planning considerations would already have an idea of what their community of people with lived experience was and therefore would be able to discern the differences between the accessibility requirements and being inclusive. 

If it needs more addressing then either a separate chapter or adjusting the accessible design vs. universal design section might be better to house a sentence pointing out that not every lived experience is a disability. 

What do you think?

Reply:
See comment below for my stab at trying to fold it in without expanding it too much!